tests/ui_tests/app/page/cluster/page.py

```diff
@@ -211,6 +220,97 @@ class ClusterConfigPage(ClusterPageMixin):
     """Cluster page config menu"""
 
     MENU_SUFFIX = 'config'
+    MAIN_ELEMENTS = [
+        ObjectPageLocators.title,
+        ObjectPageLocators.subtitle,
+        ClusterMainLocators.text,
+        ClusterConfigLocators.description_input,
+        ClusterConfigLocators.search_input,
+        ClusterConfigLocators.advanced_chbx,
+        ClusterConfigLocators.save_btn,
+        ClusterConfigLocators.history_btn,
+    ]
+
+    def fill_search_input(self, text: str):
+        self.find_element(ClusterConfigLocators.search_input).clear()
+        self.send_text_to_element(ClusterConfigLocators.search_input, text)
+
+    def click_save_btn(self):
+        self.find_and_click(ClusterConfigLocators.save_btn)
+
+    def click_history_btn(self):
+        self.find_and_click(ClusterConfigLocators.history_btn)
```

Comment:
Maybe move some of these methods to common class in common/configuration? Or are they cluster config specific?

Reply:
no, i changed tests with configuration class, thnx!